Sources/OpenTelemetrySdk/Metrics/Stable/DoubleHistogramMeterSdk.swift

```diff
@@ -1,26 +1,29 @@
 //
 // Copyright The OpenTelemetry Authors
 // SPDX-License-Identifier: Apache-2.0
-// 
+//
 
 import Foundation
 import OpenTelemetryApi
 
-public class DoubleHistogramMeterSdk : DoubleHistogram, Instrument {
+public class DoubleHistogramMeterSdk: DoubleHistogram, Instrument {
     public var instrumentDescriptor: InstrumentDescriptor
-    public var storage : WritableMetricStorage
+    public var storage: WritableMetricStorage
 
-    init(instrumentDescriptor : InstrumentDescriptor, storage: WritableMetricStorage) {
+    init(instrumentDescriptor: InstrumentDescriptor, storage: WritableMetricStorage) {
         self.instrumentDescriptor = instrumentDescriptor
         self.storage = storage
     }
     
     public func record(value: Double) {
-        record(value: value, attributes: [String:AttributeValue]())
+        record(value: value, attributes: [String: AttributeValue]())
     }
     
-    public func record(value: Double, attributes: [String : OpenTelemetryApi.AttributeValue]) {
-       //TODO: implement
+    public func record(value: Double, attributes: [String: OpenTelemetryApi.AttributeValue]) {
+        if value < 0 {
+            print("Histograms can only record non-negative values. Instrument \(instrumentDescriptor.name) has recorded a negative value.")
```

Comment:
The log message is following to Android version ([link](https://github.com/open-telemetry/opentelemetry-java/blob/ef3fef9f42d69243ac857c3e06dfc8b342d50039/sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/SdkLongHistogram.java#L40))


```
@Override
  public void record(long value, Attributes attributes, Context context) {
    if (value < 0) {
      throttlingLogger.log(
          Level.WARNING,
          "Histograms can only record non-negative values. Instrument "
              + getDescriptor().getName()
              + " has recorded a negative value.");
      return;
    }
    storage.recordLong(value, attributes, context);
  }
```